functions.php

```diff
@@ -2,11 +2,6 @@
 
 define('SIMRISHAMN_PATH', get_stylesheet_directory());
 
-// Include vendor files.
-if (file_exists(dirname(ABSPATH) . '/vendor/autoload.php')) {
-    include_once dirname(ABSPATH) . '/vendor/autoload.php';
-}
-
```

Comment:
You're right in that this probably shouldn't be here (it's in `wp-config.php` for our other projects), but I'm not sure it's included elsewhere right now. So, this might break the site?

The class loader stuff below this shouldn't be needed either if we fix the `composer.json` file in this repository. That's probably something for a different PR though.

Reply:
I had a vendor/autoload in the theme root folder in an old version. After I moved CustomModuleHelper to library/ it is now autoloaded from there and this reference is no longer needed.

Reply:
Okay, great!

Reply:
Actually, no. This is for the composer loader. You're probably thinking about [a change which was never merged](https://github.com/simrishamn/simrishamn-child/pull/36/commits/2e671d43136fbe57db43ccb6790df77feb5f94bd#diff-78cd5aa3783a74555c9938a2a81d01c6R11).

The composer autoloader has been there [since forever](https://github.com/simrishamn/simrishamn-child/commit/cf2e923302b2963ea2b1251197d188c9c53cc54b#diff-78cd5aa3783a74555c9938a2a81d01c6R5). It still works because [Municipio also includes the global autoloader](https://github.com/helsingborg-stad/Municipio/blob/4d5443f4bd0db6ac150db2b72c855988496b0319/library/Bootstrap.php#L11). I'm not so sure we should be relying on that though.

Reply:
-> @PBJ12295 

This one still needs addressing! :)